src/ResourceManager/Sql/Commands.Sql/Failover Group/Cmdlet/AddAzureSqlDatabaseToFailoverGroup.cs

```diff
@@ -0,0 +1,169 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using System;
+
+namespace Microsoft.Azure.Commands.Sql.FailoverGroup.Cmdlet
+{
+    /// <summary>
+    /// Cmdlet to add Azure Sql Databases into a Failover Group
+    /// </summary>
+    [Cmdlet(VerbsCommon.Add, "AzureRmSqlDatabaseToFailoverGroup", SupportsShouldProcess = true,
+        ConfirmImpact = ConfirmImpact.Medium)]
+    public class AddAzureSqlDatabaseToFailoverGroup : AzureSqlFailoverGroupCmdletBase
+    {
+        /// <summary>
+        /// The name of the Azure SQL Database FailoverGroup
+        /// </summary>
+        [Parameter(Mandatory = true,
+            ValueFromPipeline = true,
+            Position = 2,
+            HelpMessage = "The name of the Azure SQL Failover Group.")]
+        [ValidateNotNullOrEmpty]
+        public string FailoverGroupName { get; set; }
+
+        /// <summary>
+        /// The Azure SQL Databases to be added to the secondary server
+        /// </summary>
+        [Parameter(Mandatory = true,
+            ValueFromPipeline = true,
+            HelpMessage = "The Azure SQL Databases to be added to the secondary server.")]
+        [ValidateNotNullOrEmpty]
+        public List<AzureSqlDatabaseModel> Database { get; set; }
+
+        /// <summary>
+        /// Gets or sets the tags associated with the Azure Sql Failover Group
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "The tags to associate with the Azure Sql Elastic Pool")]
+        [Alias("Tag")]
+        public Hashtable Tag { get; set; }
+
+        /// <summary>
+        /// Overriding to add warning message
+        /// </summary>
+        public override void ExecuteCmdlet()
+        {
+            base.ExecuteCmdlet();
+        }
+
+        /// <summary>
+        /// Get the entities from the service
+        /// </summary>
+        /// <returns>The list of entities</returns>
+        protected override IEnumerable<AzureSqlFailoverGroupModel> GetEntity()
+        {
+            return new List<AzureSqlFailoverGroupModel>() {
+                ModelAdapter.GetFailoverGroup(this.ResourceGroupName, this.ServerName, this.FailoverGroupName)
+            };
+        }
+
+        /// <summary>
+        /// Create the model from user input
+        /// </summary>
+        /// <param name="model">Model retrieved from service</param>
+        /// <returns>The model that was passed in</returns>
+        protected override IEnumerable<AzureSqlFailoverGroupModel> ApplyUserInputToModel(IEnumerable<AzureSqlFailoverGroupModel> model)
+        {
+            string location = ModelAdapter.GetServerLocation(ResourceGroupName, ServerName);
+            List<AzureSqlFailoverGroupModel> newEntity = new List<AzureSqlFailoverGroupModel>();
+            List<string> dbs = new List<string>();
+
+           if (MyInvocation.BoundParameters.ContainsKey("Databases") || Database != null)
```

Comment:
@ziwa-msft this will always be true because `Databases` is a required parameter and contains the `ValidateNotNullOrEmpty` attribute